feature-libs/organization/account-summary/root/http-interceptors/blob-error.interceptor.spec.ts

```diff
@@ -34,9 +36,12 @@ describe('BlobErrorInterceptor', () => {
 
     httpMock = TestBed.inject(HttpTestingController);
     http = TestBed.inject(HttpClient);
+    windowRef = TestBed.inject(WindowRef);
   });
 
-  it(`Should extract json from errors wrapped in blob`, async () => {
+  it(`Should extract json from errors wrapped in blob`, (done) => {
```

Comment:
what do you think about adding a test case verifying the behaviour where we're running the interceptor's logic out of the browser platform?

Reply:
I had this but the test was almost the same, so I scratched it